keyboards/duck/orion/v2_5/keymaps/via/qmkvia_duckorionv25.json

```diff
@@ -0,0 +1,249 @@
+{

```

Comment:
This is the json for VIA and does not belong in this repo.  Please remove it, and PR it to `the-via/keyboards`, instead.